magma/home/__init__.py

```diff
@@ -21,14 +21,26 @@
 @home_page.route('/', methods=['GET', 'POST'])
```

Comment:
Sure, I'll leave it as is for now then.

Single line methods are what I strive for though so I'll have to start bringing you over to the dark side... ;)